cmd/ltctl/main.go

```diff
@@ -204,6 +204,38 @@ func main() {
 	}
 	rootCmd.AddCommand(collectCmd)
 
+	reportCmd := &cobra.Command{
+		Use:   "report",
+		Short: "Get or compare reports from load-tests",
+	}
+
+	genReport := &cobra.Command{
+		Use:     "generate",
+		Short:   "Generate a report from a load-test",
+		Example: "ltctl report generate --start-time=\"2020-06-17 04:37:05\" --end-time=\"2020-06-17 04:42:00\" --output=base.out --label=base",
```

Comment:
I wonder if we should use `flags` here. Flags are mostly meant to be used with default values. I mean, the command should be able to work without flags provided. Maybe we can use good old `args` here or add default values.